backend/app/services/llm/providers/sai.py

```
@@ -0,0 +1,154 @@
+from sarvamai import SarvamAI   
+
+
+
+from app.models.llm import (
+    NativeCompletionConfig,
+    LLMCallResponse,
+    QueryParams,
+    TextOutput,
+    LLMResponse,
+    Usage,
+    TextOutput,
```

Ruff (0.15.2)

[error] Redefinition of unused `TextOutput` from line 13: `TextOutput` redefined here

Remove definition: `TextOutput`

(F811)